src/migrations/20220603081324-add-archive-at-to-feature-toggle.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+        ALTER TABLE features ADD archived_at date;
```

Comment:
Should this use `TIMESTAMP WITH TIME ZONE` instead of `date`?

Reply:
Yes, that would align better with rest of the schema.